fcrepo-webapp/src/main/webapp/WEB-INF/web.xml

```diff
@@ -34,6 +34,16 @@
 
 	</servlet-mapping>
 
+  <filter>
+    <filter-name>ETagFilter</filter-name>
+    <filter-class>org.springframework.web.filter.ShallowEtagHeaderFilter</filter-class>
+  </filter>
+
+  <filter-mapping>
+    <filter-name>ETagFilter</filter-name>
+    <url-pattern>/static/*</url-pattern>
```

Comment:
Yes, `fcr:assets` also exists (and also doesn't respect ETags), but that's a completely separate issue. It's separate because `fcr:assets` is handled by `fcrepo-http-api` and not by `fcrepo-webapp` (which is what's going on here).

I would very much like to sort out the `fcr:assets` issue, but that's something of a larger deal (and relates to the LDTemplate refactoring discussed earlier).